### PR TITLE
Adds UI option for experimental DB flag.

### PR DESCRIPTION
**Pull request by zooba**
Adds UI option for experimental DB flag.
Updates Options category localized name to just "Python"

### CODE REVIEW

File: Python/Product/PythonTools/PythonTools/Options/ExperimentalOptions.cs [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+//
+// See the Apache Version 2.0 License for specific language governing
+// permissions and limitations under the License.
+
+using System;
+using EO = Microsoft.PythonTools.Interpreter.ExperimentalOptions;
+
+namespace Microsoft.PythonTools.Options {
+    public sealed class ExperimentalOptions {
+        internal ExperimentalOptions(PythonToolsService service) {
```
Comment (huguesv) [inline Python/Product/PythonTools/PythonTools/Options/ExperimentalOptions.cs]: `service` isn't used, did you leave it for possible use by later options?

File: Python/Product/PythonTools/PythonTools/Options/ExperimentalOptions.cs [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        /// <summary>
+        /// True to use the experimental feature of interpreter factories
+        /// without old-style completion databases.
+        /// </summary>
+        public bool NoDatabaseFactory { get; set; }
+
+        /// <summary>
+        /// True if the current setting is the active setting.
+        /// </summary>
+        public bool IsNoDatabaseFactoryActive => NoDatabaseFactory == EO.NoDatabaseFactory;
```
Comment (huguesv) [inline Python/Product/PythonTools/PythonTools/Options/ExperimentalOptions.cs]: This isn't used?

File: Python/Product/PythonTools/PythonTools/Options/ExperimentalOptions.cs (hunk 1 above)
Comment (zooba) [reply Python/Product/PythonTools/PythonTools/Options/ExperimentalOptions.cs]: Yeah, and consistency with the other options classes. We only have one experimental option right now and it is system-wide, but future ones could be per-instance, in which case we should use the normal options infrastructure to store the setting.

File: Python/Product/VSInterpreters/ExperimentalOptions.cs [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+// MERCHANTABLITY OR NON-INFRINGEMENT.
+//
+// See the Apache Version 2.0 License for specific language governing
+// permissions and limitations under the License.
+
+using System;
+using Microsoft.Win32;
+
+namespace Microsoft.PythonTools.Interpreter {
+    public sealed class ExperimentalOptions {
```
Comment (huguesv) [inline Python/Product/VSInterpreters/ExperimentalOptions.cs]: I think you can make the class static

File: Python/Product/PythonTools/PythonTools/Options/ExperimentalOptions.cs (hunk 2 above)
Comment (zooba) [reply Python/Product/PythonTools/PythonTools/Options/ExperimentalOptions.cs]: I was considering making the "you need to restart VS" text in the options page conditioned on this state, but decided against it. There's probably no need for this.

File: Python/Product/VSInterpreters/ExperimentalOptions.cs (hunk 3 above)
Comment (zooba) [reply Python/Product/VSInterpreters/ExperimentalOptions.cs]: 👍

**Comment by huguesv** [comment]
👍

**Pull request merged**